CHANGELOG.md

```diff
@@ -9,6 +9,8 @@
 - Print a readable error instead of dumping the whole error object to the console [#950](https://github.com/bigcommerce/cornerstone/pull/950)
 - Fixed homepage featured products floating left and unecessarily wrapping to next row [#948](https://github.com/bigcommerce/cornerstone/pull/948)
 - Add google recaptcha v2 support to cornerstone. [#951](https://github.com/bigcommerce/cornerstone/pull/951)
+- Only load amp-analytics module when an analytics ID is specified. [#957](https://github.com/bigcommerce/cornerstone/pull/957)
```

Comment:
Not sure we need new change log entry for for changes to a feature which is not yet released. We should just let all of this roll into the parent entry.